src/broker/Data.cc

```diff
@@ -694,7 +694,7 @@ struct type_checker {
         else if ( type->Tag() == TYPE_OPAQUE ) {
             // TODO: Could avoid doing the full unserialization here
             // and just check if the type is a correct match.
-            auto cpy = a;
+            const auto& cpy = a;
             auto ov = OpaqueVal::UnserializeData(BrokerListView{&cpy});
```

Comment:
I wonder if there's more to it... `auto cpy` reads like this was intended to be a copy. If it's not needed, why not use `&a` directly?

```suggestion
            auto ov = OpaqueVal::UnserializeData(BrokerListView{&a});
```